app/services/bulk_suite_selector.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+#
+# Service object to handle the bulk assignment of suites to groups. Ensures that
+# suites are available, groups don't have suites assigned, and optionally that
+# suites belong to the appropriate draw.
+class BulkSuiteSelector
```

Comment:
should this be a form object? i think all/most of the logic for that is already here, unless i'm missing something